src/keyboard.cpp

```diff
@@ -35,12 +37,14 @@ UsbKeyboard::UsbKeyboard(bool const skipUsb):
     if(!skipUsb) {
         keyBoard_.begin();
 
+	auto config = getConfig();
```

Comment:
Can we get a real type here, please and it looks like `const` is missing.

Reply:
Replaced with the actual type that is used (currently a `shared_ptr<toml::table const>`.